ocs_ci/framework/conf/default_config.yaml

```diff
@@ -62,6 +62,7 @@ REPORTING:
   us_ds: 'US'
   ocp_must_gather_image: "quay.io/openshift/origin-must-gather"
   ocs_must_gather_image: "quay.io/ocs-dev/ocs-must-gather"
+  gather_on_deploy_failure: False
```

Comment:
I think the default should be True, it makes sense to collect logs for most of the scenarios